hadoop-plugin/src/main/groovy/com/linkedin/gradle/oozie/OozieUploadTask.groovy

```diff
@@ -21,11 +21,14 @@ class OozieUploadTask extends DefaultTask {
     fs.initialize(getURI());
 
     Path directoryPath = new Path(oozieProject.dirToUpload);
-    Path projectPath = new Path(oozieProject.uploadPath + oozieProject.projectName);
+    Path projectPath = new Path(oozieProject.uploadPath + oozieProject.projectName + "/v${getProject().version}");
 
     try {
       logger.info("Project path: ${projectPath.toString()}");
       logger.info("Directory path: ${directoryPath.toString()}");
+      if(fs.exists(projectPath)) {
```

Comment:
Ah, we need to be able to oozieUpload multiple times and have it overwrite the previous version. At least that's what I think right now - I'm not completely sure yet.  Can you cause it to overwrite? I think we want to keep the exists() method you added, even if we end up not using it.